spec/features/idv/doc_auth/how_to_verify_spec.rb

```diff
@@ -72,4 +72,199 @@
       expect(page).to have_current_path(idv_hybrid_handoff_url)
     end
   end
+
+  describe 'navigating to How To Verify from Agreement page in 50/50 state' do
+    before do 
+      allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { true }
+    end
+
+      context 'opt in false at start but true during navigation' do
+        before do
+          allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { false }
+
+          sign_in_and_2fa_user
+          complete_doc_auth_steps_before_agreement_step
+          complete_agreement_step
+        end
+
+        it 'should not be bounced back from Hybrid Handoff to How to Verify' do
+          expect(page).to have_current_path(idv_hybrid_handoff_url)
+          allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+          page.refresh
+          expect(page).not_to have_current_path(idv_how_to_verify_url)
+          expect(page).to have_current_path(idv_hybrid_handoff_url)
+        end
+      end
+
+      context 'opt in true at start but false during navigation' do
+        before do
+          allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+
+          sign_in_and_2fa_user
+          complete_doc_auth_steps_before_agreement_step
+          complete_agreement_step
+        end
+
+        it 'should be redirected to Hybrid Handoff page when opt in is false' do
+          expect(page).to have_current_path(idv_how_to_verify_url)
+          allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { false }
+          page.refresh
+          expect(page).to have_current_path(idv_hybrid_handoff_url)
+        end
+      end
+    end
+
+  describe 'navigating backwards from How to Verify page in 50/50 state' do
+    before do
+      allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { true }
+    end
+
+    context 'Going back from Hybrid Handoff with opt in disabled midstream' do
+      before do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+
+        sign_in_and_2fa_user
+        complete_doc_auth_steps_before_agreement_step
+        complete_agreement_step
+        complete_how_to_verify_step
```

Comment:
This might be out of scope for this PR, but it would be great to have some indication of whether the how_to_verify step is completed with in_person or remote selected. Maybe take a named argument for the selection?

Reply:
Agreed. It is defaulted to false, but I can explicitly pass in a value during the tests to make it explicit.